google/cloud/spanner/commit_options.h

```diff
@@ -19,6 +19,7 @@
 #include "google/cloud/spanner/version.h"
 #include "google/cloud/options.h"
 #include "absl/types/optional.h"
+#include "absl/time/time.h"
```

Comment:
See the `checkers` build failure for problems with include ordering and indentation.

Reply:
When I look at that build failure it appears empty. When I click "see more information" it looks like I don't have permission. Am I missing something?

Reply:
There are other `checkers` failures too.  Let me know if you still can't see them and we can try to fix that and/or quote them for you.